libs/core/src/lib/skeleton/components/skeleton.component.ts

```diff
@@ -0,0 +1,73 @@
+import { ChangeDetectionStrategy, Component, HostBinding, Input, ViewEncapsulation } from '@angular/core';
+import { random } from 'lodash-es';
+
+export type SkeletonType = 'circle' | 'rectangle' | 'text';
+export type SkeletonWidth = 'rand' | string;
+export type SkeletonHeight = 'auto' | string;
+
+@Component({
+    selector: 'fd-skeleton',
+    template: `<ng-content></ng-content>`,
+    styleUrls: ['skeleton.component.scss'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    // eslint-disable-next-line @angular-eslint/no-host-metadata-property
+    host: {
+        class: 'fd-skeleton'
+    }
+})
+export class SkeletonComponent {
+    /** Wight of the skeleton */
+    @Input()
+    set width(value: SkeletonWidth) {
+        if (value === 'rand') {
+            this._width = getRandomWidth();
+            return;
+        }
+
+        this._width = value;
+    }
+
+    /** Height of the skeleton. Relevant if type 'circle' or 'rectangle'. */
```

Comment:
If the type is circle, is it really needed to use height and width? It's kinda round, so it either one of those values that needs to be used (not sure if we should support ellipses)